src/gasPriceOracle/oracle.ts

```diff
@@ -17,22 +17,28 @@ import * as polygonViem from "./adapters/polygon-viem";
  * @param chainId The chain ID to query for gas prices.
  * @param provider A valid ethers provider.
  * @param legacyFallback In the case of an unrecognised chain, fall back to type 0 gas estimation.
+ * @parm baseFeeMarkup Multiplier applied to base fee for EIP1559 gas prices (or total fee for legacy).
  * @returns Am object of type GasPriceEstimate.
  */
 export async function getGasPriceEstimate(
   provider: providers.Provider,
   chainId?: number,
+  baseFeeMarkup = 1.0,
```

Comment:
Not sure about the name `baseFeeMarkup` since it's actually a multiplier, not a markup. Consider `baseFeeMultiplier` instead?